src/collectors/file_collector.cpp

```diff
@@ -1,5 +1,4 @@
-#include "file_collector.hpp"
-
+template <typename MsgType>
```

Comment:
This file can be safely deleted, correct?

Reply:
sure, Idk why it's till here, I have messed up